python/cuml/dask/ensemble/randomforestclassifier.py

```diff
@@ -136,25 +142,37 @@ def __init__(
         super(RandomForestClassifier, self).__init__(client=client,
                                                      verbose=verbose,
                                                      **kwargs)
+        if seed is not None:
+            if random_state is None:
+                warnings.warn("Parameter 'seed' is deprecated and will be"
+                              " removed in 0.17. Please use 'random_state'"
+                              " instead.",
```

Comment:
Text should reflect that `random_state` is being set as opposed to ask the user to set them instead.

Reply:
Good catch. I'll probably update this the `train_test_split` warning as well (since I took this from there), even though its unrelated to this PR.

https://github.com/rapidsai/cuml/blob/f35eb5a166d84f3f8b57a0002aa6770622ffc783/python/cuml/preprocessing/model_selection.py#L330-L332